src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -1397,7 +1397,6 @@ public function providePathsForIsAbsolutePath()
             ['var/lib', false],
             ['../var/lib', false],
             ['', false],
-            [null, false],
```

Comment:
Why do we have to remove this one?

Reply:
`Filesystem::isAbsolutePath` accepts only strings.

Reply:
then we should deprecate passing `nulll` in 4.4 first

Reply:
Okay, I will provide a PR

Reply:
https://github.com/symfony/symfony/pull/32315